crawler/src/main/java/org/iqu/crawler/configuration/ConfigurationManager.java

```diff
@@ -0,0 +1,44 @@
+package org.iqu.crawler.configuration;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+
+
+public class ConfigurationManager {
+	
+	private Properties prop = new Properties();
+	private InputStream input = null;
+	private List<SourceConfiguration> properties = new ArrayList<SourceConfiguration>();
+
+	public void loadFile() {
+		try {
+			input = new FileInputStream("src/main/resources/configuration.properties");
```

Comment:
Nu facem teste in main. Veau varianta finala in pull request. Nu are ce sa caute o resursa aici